packages/blade/src/components/Checkbox/CheckboxIcon/Fade.native.tsx

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+import React from 'react';
+import type { EasingFn } from 'react-native-reanimated';
+import Animated, { Keyframe } from 'react-native-reanimated';
+import type { CSSObject } from 'styled-components';
+import styled from 'styled-components/native';
+import { useTheme } from '../../BladeProvider';
+
+type FadeProps = {
+  children: React.ReactNode;
+  show?: boolean;
+  styles: CSSObject;
+};
+
+const StyledFade = styled(Animated.View)<{ styles: CSSObject }>(({ styles }) => {
+  return {
+    ...styles,
+  };
+});
+
+const Fade = ({ children, show, styles }: FadeProps) => {
+  const { theme } = useTheme();
+
+  const fadeInEasing = (theme.motion.easing.entrance.effective as unknown) as EasingFn;
+  const fadeOutEasing = (theme.motion.easing.exit.effective as unknown) as EasingFn;
```

Comment:
this is exactly what i've commented above  for writing fadeIn and fadeOut as part of `Fade` file itself.

Reply:
Yes done.